administrator/components/com_users/models/mail.php

```diff
@@ -162,8 +162,8 @@ public function send()
 
 		// Build email message format.
 		$mailer->setSender(array($app->get('mailfrom'), $app->get('fromname')));
-		$mailer->setSubject($params->get('mailSubjectPrefix') . stripslashes($subject));
-		$mailer->setBody($message_body . $params->get('mailBodySuffix'));
+		$mailer->setSubject($params->get('mailSubjectPrefix') . " " . stripslashes($subject));
```

Comment:
```suggestion
		$mailer->setSubject($params->get('mailSubjectPrefix') . ' ' . stripslashes($subject));
```
Don't use double quotes.